Containers/apache/Dockerfile

```diff
@@ -1,59 +1,44 @@
 # Caddy is a requirement
 FROM caddy:2.6.2-alpine as caddy
 
-FROM debian:bullseye-20221219-slim
+FROM httpd:2.4.54-alpine3.17
```

Comment:
lets wait for alpine 3.17.1 as well

Reply:
the httpd image was now pushed to alpine v3.17.1

Reply:
It looks like I will not get to this due to time problems before the next beta release which is planned for coming tuesday. Sorry for that! I'll try to include it in the version afterwards.